Resources/Prototypes/_Impstation/Entities/Mobs/Customization/Markings/misc.yml

```
@@ -0,0 +1,903 @@
+    state: chitinidbeetlehorn1
+
+- type: marking
+  id: ChitinidBeetleHorn2
+  bodyPart: HeadTop
+  markingCategory: HeadTop
+  speciesRestriction: [Chitinid]
+  forcedColoring: true
+  followSkinColor: true
+  sprites:
+  - sprite: _Impstation/Mobs/Customization/markingsbundle1.rsi
+    state: chitinidbeetlehorn2
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)